cmd/scaffold.go

```diff
@@ -461,18 +445,20 @@ func (fnb *FlowNodeBuilder) InitFlowNetworkWithConduitFactory(
 	}
 
 	// creates network instance
-	net, err := p2p.NewNetwork(&p2p.NetworkConfig{
-		Logger:              fnb.Logger,
-		Codec:               fnb.CodecFactory(),
-		Me:                  fnb.Me,
-		SporkId:             fnb.SporkID,
-		MiddlewareFactory:   func() (network.Middleware, error) { return fnb.Middleware, nil },
-		Topology:            topology.NewFullyConnectedTopology(),
-		SubscriptionManager: subscriptionManager,
-		Metrics:             fnb.Metrics.Network,
-		IdentityProvider:    fnb.IdentityProvider,
-		ReceiveCache:        receiveCache,
-		ConduitFactory:      cf,
+	net, err := p2pnet.NewNetwork(&p2pnet.NetworkConfig{
```

Comment:
```suggestion
	net, err := p2pnet.NewEngineNetwork(&p2pnet.EngineNetworkConfig{
```

Reply:
The network serves as a comprehensive layer that integrates three interfaces within Flow. Renaming it as `NewNetworkEngine` would misrepresent its function, implying it's merely an engine component rather than an all-encompassing networking layer.